BusinessTripService/ManageBTDB/Program.cs

```diff
@@ -0,0 +1,233 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using ManageBTDB.CRUD;
+using ManageBTDB.Service;
+
+
+namespace ManageBTDB
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            ServiceReporter.ShowServiceMessage("ЧТЕНИЕ ТАБЛИЦЫ НОРМ РАСХОДОВ ПО ПРОЕЗДУ:");
```

Comment:
Things like this are usually done with unit tests* - it's much easier to check if values are as you expect them.

*this kind of tests is called integration test, as they test integration between your code and DB. but they're implemented using unit test frameworks.